### PR TITLE
added support page with text and links

### PR DESCRIPTION
**Pull request by kserrec**
### Description

First changes were made by edswf1, see - https://github.com/edswf1/dogecoin.com-1 which added a support html page with empty cards to be filled in with content. I wrote the content that filled them in, added buttons with links to relevant material, and added a way to get to the support page from home.

### Motivation and Context

Issue # 138

### How Has This Been Tested?

Manual testing

### Checklist:

- [x] GitHub issue linked - (Not applicable for this PR, we don't want this to close the issue)
- [ ] The necessary translations have been added for all languages
- [ ] Any documentation has been updated accordingly
- [ ] Responsive sizes (Mobile) tested
- [ ] Cross Browser tested if necessary
- [x] Conflicts resolved

### CODE REVIEW

**Comment by edswf1** [comment]
if you need any help regarding the structure just let me know

File: support.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            margin: 0px auto;
+            padding: 20px;
+            border-radius: .5rem;
+        }
+        .support-content .fa{
+            font-size: 2rem;
+        }
+        .support-content .heading{
+            font-size: 1.2rem;
+        }
```
Comment (edswf1) [inline support.html]: i inlined these for my first draft -> i think they should be moved to styles.css

File: index.html [hunk 2]
```diff
@@ -46,6 +46,7 @@
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#about"><span data-i18n="dgc-what-is-doge">What is dogecoin</span></a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#start"><span data-i18n="dgc-get-started-now">How to get started</span></a></li>
                     <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="guide.html"><span data-i18n="dgc-guide">Guide</span></a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="support.html"><span data-i18n="dgc-support">Support</span></a></li>
```
Comment (patricklodder) [inline index.html]: Could you remove this for now? Then we just merge to the main branch and publish the page, but leave it "orphaned" until we have more translations in and tuned the text.

File: support.html (hunk 1 above)
Comment (patricklodder) [reply support.html]: seconded.

**Review by patricklodder — CHANGES_REQUESTED** [review]
I have marked minimal changes needed for immediate merge + publication. If you change those, we can merge and the subsequent process would be:

1. Fine-tune wording and complement with any additional items we'd want to highlight.
2. Arrange for translations
3. Integrate fully (=link the page from index.html)
4. 🚀

**Comment by kserrec** [comment]
I tried to address both comments. I'm not sure why the change to styles.css caused a conflict or how to fix it; it's just tacked on styles at the end.

**Comment by kserrec** [comment]
As for these: 
    `1. Fine-tune wording and complement with any additional items we'd want to highlight.
     2 Arrange for translations`

I have found people who have agreed to translate the support page into Spanish and German. No French yet. But I'm wondering if I should wait on the fine-tuning of the wording before having them do the translation or if there are already people you guys have in mind.

**Review by patricklodder — DISMISSED** [review]
All look good to me! Thank you!

**Review by patricklodder — APPROVED** [review]

**Review by edswf1 — APPROVED** [review]

**Review by AnthonySaldana — APPROVED** [review]

**Pull request merged**